app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -3,8 +3,40 @@ kind: registration
 name: Global Registration
 model: ProvisioningTemplate
 -%>
+<%
+    curl_opts = ["-H 'Content-Type: application/json'", "-H 'Accept: application/json'", "-H 'Authorization: Bearer #{@auth_token}'"]
+%>
+#!/bin/bash
+
+# Template parameters:
+<%= "#  - User: [#{@user.login}]" if @user %>
+<%= "#  - Organization: [#{@organization.name}]" if @organization %>
+<%= "#  - Location: [#{@location.name}]" if @location %>
+<%= "#  - Hostgroup: [#{@hostgroup.name}]" if @hostgroup %>
 
 if ! [ $(id -u) = 0 ]; then
     echo "Please run as root"
     exit 1
 fi
+
+OS=$(cat /etc/os-release | awk -F '=' '/^ID=/ {print $2}')
```

Comment:
Isn't it easier to just source it? Also note that on non-Linux this file doesn't exist.
```suggestion
if [ -f /etc/os-release ] ; then
  . /etc/os-release
fi
```